src/ParametersHandler/include/BipedalLocomotion/ParametersHandler/IParametersHandler.h

```diff
@@ -192,6 +193,12 @@ class IParametersHandler
      */
     virtual std::string toString() const = 0;
 
+    /**
+     * Return the names of all the stored parameters.
+     * @return A set containing the names of all stored parameters.
+     */
+    virtual std::unordered_set<std::string> getParametersNames() const = 0;
```

Comment:
Just a curiosity, why you chose this specific container?

Reply:
I think among all the data structures supported in the STL this is the best fit, if I understood the names of the parameters should be unique and there's no reason I can see to know their order.

Reply:
OK makes sense!